### PR TITLE
add colofilter.css@1.0.0 w/ git auto-update config

### LINKED ISSUE #7826: [Request] Add colorfilter
**Library name:** colorfilter
**Git repository url:** https://github.com/LukyVj/colofilter.css
**npm package url(optional):** 
**License(s):**
**Official homepage:** https://lukyvj.github.io/colofilter.css/
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by kennynaoh**
PR for #7826 
@x09326 please help me review this pull request, thank you.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/LukyVj/colofilter.css
 * Official website (optional, not the repository): https://lukyvj.github.io/colofilter.css/
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 427
   - Count of watchers: 11
   - Count of forks: 38
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #7826, cc @LukyVj

### CODE REVIEW

File: ajax/libs/colofilter.css/package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "css"
+  ],
+  "author": "Lucas Bonomi",
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/LukyVj/colofilter.css.git",
+    "basePath": "styles",
+    "files": [
+      "colofilter*.!(scss)"
```
Comment (x09326) [inline ajax/libs/colofilter.css/package.json]: @kennynaoh 
I'm not sure whether we need other files.
Can you confirm this with the author?

Comment (kennynaoh) [inline ajax/libs/colofilter.css/package.json]: What you mean about other files?

Comment (LukyVj) [inline ajax/libs/colofilter.css/package.json]: You just need that.

**Comment by x09326** [comment]
@PeterDaveHello 
I think this PR is okay, please help me review it, thanks.

File: ajax/libs/colofilter.css/package.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "colofilter",
+    "duotone",
+    "css"
+  ],
+  "author": "Lucas Bonomi",
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/LukyVj/colofilter.css.git",
+    "basePath": "styles",
```
Comment (PeterDaveHello) [inline ajax/libs/colofilter.css/package.json]: `styles` is not a good base path.

Comment (LukyVj) [inline ajax/libs/colofilter.css/package.json]: Here is the file https://raw.githubusercontent.com/LukyVj/colofilter.css/master/styles/colofilter.css

**Comment by kennynaoh** [comment]
@PeterDaveHello 
So, what should I do next?
Thanks!

**Comment by PeterDaveHello** [comment]
As I said, you should not use it as base path.

**Comment by kennynaoh** [comment]
`"basepath" : "",`
`"files":"styles/colofilter*"`
Is this better?
Thanks!

**Comment by PeterDaveHello** [comment]
Yes.

**Comment by kennynaoh** [comment]
@x09326 
I have fixed the `basePath`
Please help me review it again.
Thanks!

**Pull request merged**